### ISSUE TITLE
Gotenberg - Ingress

### ISSUE THREAD

**Issue by TomKeur** [Kurt108/helm-charts#31]
Thanks for the Gotenberg Helm Chart 🎉

When installing the Helm Chart, I'm getting the following warning:

```
W0907 20:41:41.853703   46905 warnings.go:70] extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
W0907 20:41:42.009434   46905 warnings.go:70] extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
```

If you want I can create an PR and update the Ingress template.

**Comment by Kurt108** [comment]
Hi Tom,

Yes, that would be awesome!

Kurt

**Comment by Kurt108** [comment]
I'm sorry for the delay. I want to update the chart to gotenberg7 this weekend and will merge your PR together with that task

**Comment by Kurt108** [comment]
fixed with gotenberg-4.0.1
